src/node_modules/react/lib/getNextDebugID.js

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
Yea. This is correct. When we switch to lerna or yarn workspaces, this will be generated by tooling but since we don't have the tooling in place yet, I just checked in forwarding modules.